### PR TITLE
Fix shearlab invadj

### PR DESCRIPTION
**Pull request by arsenal9971**
Here is the cherry picking of the fix

### CODE REVIEW

**Comment by arsenal9971** [comment]
@kohr-h I have one big question. I am working right now on the inversion of a composition between two operators (the ray transform and the inverse shearlet transform). The composition via `python odl.operator.operator.OperatorComp`

File: odl/contrib/shearlab/shearlab_operator.py [hunk 1]
```diff
@@ -171,8 +179,50 @@ def _call(self, x):
...
+                    of shearlet transform.
+
+                    See Also
+                    --------
+                    odl.contrib.shearlab.ShearlabOperator
+                    """
+
+                    def __init__(self):
+                        """Initialize a new instance.
+                        Parameters
```
Comment (aringh) [inline odl/contrib/shearlab/shearlab_operator.py]: This `__init__` method seems to take no parameters as input. So the docstring could just be `"""Initialize a new instance."""`. This also goes for the methods on [line 78](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L78), [line 117](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L117), and [line 164](https://github.com/arsenal9971/odl/blob/5e762cb9ed7cb406591bb1adae5496f2b62339f2/odl/contrib/shearlab/shearlab_operator.py#L164).

Comment (arsenal9971) [reply odl/contrib/shearlab/shearlab_operator.py]: Right, let me commit tomorrow this fixes, sorry for my lack proficiency on git, I will cherry pick tomorrow

Comment (aringh) [reply odl/contrib/shearlab/shearlab_operator.py]: No worries. Thank you for great contributions to `odl`! :smile:

**Comment by arsenal9971** [comment]
I am not sure if the cherry picking is working

**Comment by kohr-h** [comment]
> I am not sure how to do the cherry pick again. But this is the last fix so far I am gonna do.

Is the `Adding inverse adjoint of the shearlet operator correctly` commit the first one you'd like to have? I can do the cherry-picking for you and give you instructions on how to update your branch.

**Comment by sbanert** [comment]
I’m just interesting: I see myself as an author of commit `5e762cb` (https://github.com/odlgroup/odl/pull/1358/commits/5e762cb9ed7cb406591bb1adae5496f2b62339f2), which confuses me a bit.

**Comment by arsenal9971** [comment]
@sbanert That is weird, I am also not aware you did that changes

**Comment by arsenal9971** [comment]
@kohr-h What do you mean with the local name of the remote of the fork?, I think it is arsenal9971 (https://github.com/arsenal9971/odl), when I am doing the last step 
```
git push -f <arsenal7791> kohr-h/shearlab_inv_adj_new:fix_shearlab_invadj
```
but I got the error
```
fatal: 'arsenal9971' does not appear to be a git repository
fatal: Could not read from remote repository.
```

**Comment by aringh** [comment]
@arsenal9971: `git remote -v` will give you a list of the names of all remotes you have in your repository. For me this gives
```
my_fork	https://github.com/aringh/odl.git (fetch)
my_fork	https://github.com/aringh/odl.git (push)
origin	https://github.com/odlgroup/odl.git (fetch)
origin	https://github.com/odlgroup/odl.git (push)
sbanert	https://github.com/sbanert/odl.git (fetch)
sbanert	https://github.com/sbanert/odl.git (push)
```

**Comment by arsenal9971** [comment]
@aringh For some weird reason I get:
```
kohr-h	https://github.com/kohr-h/odl (fetch)
kohr-h	https://github.com/kohr-h/odl (push)
origin	git@github.com:arsenal9971/odl.git (fetch)
origin	git@github.com:arsenal9971/odl.git (push)
upstream	git@github.com:odlgroup/odl.git (fetch)
upstream	git@github.com:odlgroup/odl.git (push)
```

and when I am trying to add my own remote `git remote add origin git@github.com:arsenal9971/odl.git`, I get `fatal: remote origin already exists.`

**Comment by kohr-h** [comment]
> Now is move here #1363.

Actually this PR is clean now, so you don't need the new one. (The test failure is random, we need to fix that separately.)

File: odl/contrib/shearlab/shearlab_operator.py (hunk 1 above)
Comment (kohr-h) [reply odl/contrib/shearlab/shearlab_operator.py]: Remaining TODO

**Comment by kohr-h** [comment]
Closing in favor of #1363

**Pull request closed without merging**